modules/doc/training/content/sqa/training/ccb/pln-4005.md

```diff
@@ -0,0 +1,187 @@
+# PLN-4005
+
+## Software Quality Assurance Plan (SQAP) for MOOSE and MOOSE-based Applications
+
+!---
+
+## PLN-4005: Purpose
+
+- Static document created to detail the process for development of MOOSE and MOOSE-based
+  applications to meet the NQA-1 standard using [!ac](INL) templates and procedures.
+- +Required+ reading associated with this training
+
+!---
+
+## PLN-4005: Assumptions
+
+- MOOSE and MOOSE-based applications are designated as Quality Level 2 (QL-2) non-safety software
+  but are managed following processes that implement Quality Level 1 (QL-1) safety software
+- In short, we follow the most stringent processes but do not provide software at that level, it is
+  necessary for the end-users to complete the process before using our software in a safety
+  capacity
+
+!---
+
+## PLN-4005: Roles
+
+- +Line Manager+: Manages the overall project ensuring that work scope is defined and adequate
+  resources exist (time, talent, funding, etc.)
+- +Project Lead+: Manages day-to-day development of the code to ensure that work is being performed
+  as defined by the Line Manager, and performs software releases
+- +Software Quality Assurance Lead+: Manages [!ac](SQA) processes and ensures that PLN-4005 is
+  being implemented
+- +Change Control Board (CCB)+: Perform reviews of Change Requests (CRs) to ensure that code
+  changes are necessary and that documentation and testing is performed as defined in PLN-4005
+
+!---
+
+## PLN-4005: Documentation
+
+Three classes of documentation are defined for MOOSE and MOOSE-based applications:
+
+- static
+- repository
+- development
+
+!---
+
+### PLN-4005: Static Documentation
+
+- +Software Quality Assurance Plan (SQAP; PLN-4005)+: Defines [!ac](SQA)
+  processes implemented including Configuration Management Plan (CMP) and Verification
+  and Validation Plan (VVP)\*.
+- +Safety-software Determination (SSD)+: [!ac](INL) form to declare if the application is safety
+  software
+- +Quality-level Determination (QLD)+: [!ac](INL) form to declare the safety software level
+- +[!ac](EA) Entry+: [!ac](INL) from that lists [!ac](SQA) documentation for application
```

Comment:
No form #s for these?

Reply:
Not that I can find, the are online sites